client/state/reader-ui/sidebar/reducer.js

```diff
@@ -45,14 +45,14 @@ export const openOrganizations = withPersistence( ( state = [], action ) => {
 	return state;
 } );
 
-export const selectedRecentSite = withPersistence( ( state = null, action ) => {
+export const selectedRecentSite = ( state = null, action ) => {
```

Comment:
It no longer seems necessary to have persistence for this or bloat the persisted data, since the individual route will communicate the selection.